networkx/algorithms/centrality/dispersion.py

```diff
@@ -19,6 +19,9 @@ def dispersion(G, u=None, v=None, normalized=True, alpha=1.0, b=0.0, c=0.0):
         The target of the dispersion score if specified.
     normalized : bool
         If True (default) normalize by the embededness of the nodes (u and v).
+    alpha, b, c : float
+        If normalized is True (default), try out different values of alpha, b and, c to obtain maximum
+        performance of normalized dispersion.
```

Comment:
In this case I think the best thing to do would be to explicitly describe how the parameters are used in the embeddedness normalization.

Reply:
" To strengthen the performance of a normalized dispersion, the nodes can be ranked by the order of the function, (disp(u, v) + b)α/(emb(u, v) + c). Searching over choices of alpha, b, and, c leads to maximum performance." How about this?

Reply:
Great. Thanks for your help!